### PR TITLE
DOC: add example showing how to convert POSIX timestamps to datetime64

### PR DESCRIPTION
**Pull request by jonathangjertsen**

### CODE REVIEW

**Comment by jonathangjertsen** [comment]
I accidentally committed my text editor's removal of trailing whitespace. Let me know if I should amend the commit to undo it

File: doc/source/reference/arrays.datetime.rst [hunk 1]
```diff
@@ -70,6 +70,18 @@ datetime type with generic units.
     array(['2001-01-01T12:00:00.000', '2002-02-03T13:56:03.172'],
           dtype='datetime64[ms]')
 
+An array of datetimes can be constructed from integers representing
+POSIX timestamps with the given unit.
+
+.. admonition::Example
```
Comment (melissawm) [inline doc/source/reference/arrays.datetime.rst:76]:
```suggestion
.. admonition:: Example
```

**Review by melissawm — CHANGES_REQUESTED** [review]
Hi @jonathangjertsen - thanks for that! As you can see in the generated [CircleCI artifact](https://19788-908607-gh.circle-artifacts.com/0/doc/build/html/reference/arrays.datetime.html), your example doesn't show up - you are missing a space before the word "Example". Otherwise, looks good to me!

It's a good idea to [build the docs locally](https://numpy.org/devdocs/docs/howto_build_docs.html) when you do a change and check that it all works, let me know if you need help with that.

**Comment by jonathangjertsen** [comment]
> It's a good idea to [build the docs locally](https://numpy.org/devdocs/docs/howto_build_docs.html) when you do a change and check that it all works, let me know if you need help with that.

Thanks, I'll make sure to do that next time

**Review by melissawm — APPROVED** [review]

**Comment by melissawm** [comment]
Thanks, @jonathangjertsen !

**Pull request merged**